### PR TITLE
fix: fixes based on feedback

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sources/google-tag-manager-client.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Note: We do *not* recommend installing amplitude through a Custom HTML tag.
  
 #### API Key
 
-Copy your amplitude API Key in the API Key field. For EU residency, please make sure your API Key is under analytics.eu.amplitude.com.
+Copy your amplitude project API Key in the API Key field. For EU residency, your project API Key is under analytics.eu.amplitude.com. Each project has different API Key, please make sure you are copy the API Key from the right project. Go to **Settings -> Projects -> click the right project name from the list** to find your project API Key.
```
Comment (justin-fiedler) [inline docs/data/sources/google-tag-manager-client.md:42]: Maybe make `analytics.eu.amplitude.com` a link? Or call out using "code style" with tikmarks?

File: docs/data/sources/google-tag-manager-client.md [hunk 2]
```diff
@@ -170,6 +170,9 @@ Set event level groups. With event-level groups, the group designation applies o
 
 #### identify
 
+!!!note 
+    Identify calls will not be showing like a single event in user look up. The identify calls will set the user properties, and those updated user properties will appear only after the next event fired by the User.
```
Comment (justin-fiedler) [inline docs/data/sources/google-tag-manager-client.md:174]:
```suggestion
    Identify calls _don't_ appear in user look up. The identify calls will set the user properties, and those updated user properties will appear only after the next event fired by the User.
```

**Review by justin-fiedler — APPROVED** [review]
Overall LGTM. See nits.

File: docs/data/sources/google-tag-manager-client.md (hunk 1 above)
Comment (Mercy811) [reply docs/data/sources/google-tag-manager-client.md:42]: How about adding the link https://www.docs.developers.amplitude.com/analytics/find-api-credentials/?

File: docs/data/sources/google-tag-manager-server.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Create tags for your amplitude browser SDK tracking. Click the `New` button to c
 
 ##### API Key
 
-Copy your amplitude API Key in the API Key field. For EU recidency, please make your API Key is under analytics.eu.amplitude.com.
+Copy your amplitude project API Key in the API Key field. For EU residency, your project API Key is under analytics.eu.amplitude.com. Each project has different API Key, please make sure you are copy the API Key from the right project. Go to **Settings -> Projects -> click the right project name from the list** to find your project API Key.
```
Comment (Mercy811) [inline docs/data/sources/google-tag-manager-server.md:43]: Maybe make an include file to share with client docs?

**Review by Mercy811 — APPROVED** [review]
Thanks Alyssa for taking feedback and improving the doc🙏

**Pull request merged**